src/services/registries-search-services.ts

```diff
@@ -32,29 +32,28 @@ export default class BusinessLookupServices {
     let url = this.searchApiUrl + 'businesses/search/facets?start=0&rows=20'
     url += `&categories=legalType:${legalTypes}${status ? '::status:' + status : ''}`
     url += `&query=value:${encodeURIComponent(query)}`
-
-    const kcToken = sessionStorage.getItem(SessionStorageKeys.KeyCloakToken)
-
+    // const kcToken = sessionStorage.getItem(SessionStorageKeys.KeyCloakToken)
     const config = {
       headers: {
-        Authorization: `Bearer ${kcToken}`,
+        // Authorization: `Bearer ${kcToken}`,
```

Comment:
I'm just testing if this is not needed (since it's added by an interceptor anyway).